From 51c2d666630dc1450e32a27d023a45dd62d3b49d Mon Sep 17 00:00:00 2001
From: Petr Vobornik <pvoborni@redhat.com>
Date: Tue, 20 Nov 2012 18:08:57 +0100
Subject: [PATCH] WebUI: Change of default value of type of new group back to
 POSIX

In FreeIPA 2.2, the default for group type was 'POSIX', in 3.0 it's 'normal' (non-posix). CLI's default is 'POSIX', Web UI should use 'POSIX' as well.

https://fedorahosted.org/freeipa/ticket/3270
---
 install/ui/group.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install/ui/group.js b/install/ui/group.js
index 875e67f027808d373a38e6b32964f111fe22a94b..8317d679fcc682126081db0d55fcb89f621b8413 100644
--- a/install/ui/group.js
+++ b/install/ui/group.js
@@ -153,7 +153,7 @@ IPA.group.entity = function(spec) {
                     name: 'type',
                     label: IPA.messages.objects.group.type,
                     flags: ['no_command'],
-                    default_value: 'normal',
+                    default_value: 'posix',
                     options: [
                         {
                             value: 'normal',
-- 
1.7.11.7

